kubespawner/proxy.py

```diff
@@ -44,6 +44,90 @@ def endpoints(self):
 
 
 class KubeIngressProxy(Proxy):
+    """
+    DISCLAIMER:
+
+        This class is not maintained thoroughly with tests and documentation, or
+        actively used in any official distribution of JupyterHub.
+
+        When it was originally developed and piloted by Yuvi (@yuvipanda), it is
```

Comment:
Ah, I think the understanding instead should be that the default chp was good enough for the eventual traffic circumstances I ended up having (~2k users, not 20k users). It was not less reliable or responsive - just that the chp level of reliability was 'good enough'

Reply:
Ah! I rewrote this section!